cargo-{deb, msrv, binstall}: add notice to indiacate non-standard command (#22275)
tldr-pages/tldr: pages/common/cargo-deb.md

# cargo deb

> Create Debian packages from Cargo projects.

> > Note: This is not a built-in Cargo command; install it first with `cargo install cargo-deb`.

Reviewer: You can also install this with other package managers.
```suggestion
> Note: This is not a built-in Cargo command, you need to install it first.
```

Author: I agree with you, but `cargo install cargo-deb` is the installation method recommended by [cargo-deb](https://github.com/kornelski/cargo-deb). In fact, it's the only installation method stated in that project's README. Since this is a "tl;dr" page, perhaps it would be better to point out the most obvious installation method directly instead of making users click the "More info" link to look for how to install it?

Reviewer: > perhaps it would be better to point out the most obvious installation method

*Is it* the most obvious installation method? I would say the first thing people try is the distribution's official package manager. Besides, we don't document installation methods - I don't think there's a single tldr page that does.

Author: fixed
